src/renate/memory/buffer.py

```diff
@@ -201,7 +200,12 @@ def set_transforms(
         self._target_transform = target_transform
 
     def to_tensors(self) -> Tuple[Tensor, ...]:
```

Comment:
This might lead to errors for _general_ inputs. `Buffer._data_points` is a tuple `(inputs, outputs)`, where `outputs` is a `torch.Tensor`, but `inputs` may be `torch.Tensor` or a tuple/dict thereof. I guess that the avalanche learners only support the simple `(X, y)` case? If so, we should put a check here, making sure that `inputs` is `torch.Tensor`.

Reply:
will this be a problem as long as we won't change the `ModelUpdater` for training a model?

Reply:
I don't understand your comment. My point is: If your `Buffer` contains general inputs, say `buffer[0] = (x, y)` with `x = {"token_ids": aaa, "attention_mask": bbb}` and you call `Buffer.to_tensors()`, this will lead to an error. It would be easy to make the method more general by calling it recursively.